This is the real issue

```06:10:01 2019-07-09 04:10:01.765 SourceKitLSPPackageTests.xctest[2773:38ff9700] manifest parse error(s):
06:10:01 
/home/buildnode/jenkins/workspace/swift-PR-Linux@2/branch-master/tmp/TemporaryDirectory.QLRGsf/pkg/Package.swift:3:5: error: type annotation missing in pattern
06:10:01 let pack
06:10:01     ^```

@aciidb0mb3r / @benlangmuir / @akyrtzi 

is this something you're aware of?

--- a/utils/build-script-impl
+++ b/utils/build-script-impl
@@ -3555,8 +3555,6 @@ for host in "${ALL_HOSTS[@]}"; do
                             continue
                         ;;
                     esac
-                else
-                    INSTALL_TARGETS="install-lldb"
                 fi
                 ;;
             swiftpm)

